content/tutorials/modeling_movement/GRASS_movement.qmd

```diff
@@ -0,0 +1,565 @@
+---
+title: "Modeling Movement in GRASS"
+author: "Michael Barton & Anna Petrosova"
+date: 2025-02-10
+date-modified: today
+format: 
+  html:
+    embed-resources: true
+    toc: true
+    code-tools: true
+    code-copy: true
+    code-fold: false
+page-layout: article
+categories: [intermediate, advanced, GUI, raster, cost surface, least cost path]
+description: Generating a cumulative cost surface and least cost path with *r.walk* and *r.path* to model movement by walking across a landscape.
+engine: jupyter
+execute: 
+  eval: false
+Jupyter: python3
+
+# © Michael Barton & Anna Petrosova 2025
+---
+
+GRASS has sophisticated tools to model movement across terrain, including [***r.cost***](https://grass.osgeo.org/grass-stable/manuals/r.cost.html), [***r.walk***](https://grass.osgeo.org/grass-stable/manuals/r.walk.html), [***r.drain***](https://grass.osgeo.org/grass-stable/manuals/r.drain.html), and [***r.path***](https://grass.osgeo.org/grass-stable/manuals/r.path.html). In this tutorial, we will use ***r.walk*** and ***r.path*** to determine the best walking path to reach a destination, like a hospital.
+
+::: {.callout-note title="Dataset"}
+This tutorial uses one of the standard GRASS demo data: flagstaff_arizona_usa. We will refer to place names in that data set, but it can be completed with any of the standard demo data sets for any region. We will use the *elevation* DEM, the *hospitals* vector points file (any other vector point file will serve), and the *landuse* raster map.
```

Comment:
Can we have a link to the dataset? Where can people find it?

Reply:
We can't until GRASS makes the data accessible on its web site

Reply:
If we put it into a temporary location, the link will be invalid when the dataset is posted to the web site

Reply:
I solved this like that previously:
https://github.com/petrasovaa/grass-tutorials/blob/d31e4d187b53938b9cecace9da66d16c496f7c52/content/tutorials/least_cost_path_r_walk/least_cost_path_r_walk.qmd#L31

Reply:
Yes. That is a temporary location that will need to change. And it is openly available to anyone without logging in? I don't mind if you want to use it. The idea of this and other tutorials is that they could be done with any data set that has GRASS standard demo data.

Reply:
I am sorry I don't understand your last comment, what temporary location do you mean?

Reply:
The Flagstaff date set is currently located on my and your Google Drives. That is not where it will be eventually. And I'm not even sure all users would have access to the data in those locations. So I did not want to make a link to the Flagstaff data at either place.